modules/cnv_install_1_4.adoc

```diff
@@ -17,7 +17,12 @@ This procedure installs the following components:
 * A running {product-title} 3.11 cluster
 * User with *cluster-admin* privileges
 
-// add prereqs for remote installation
+[NOTE]
+====
+If you execute the playbook from a remote host, that host must have the 
```

Comment:
I don't see the point of this note. kubevirt-ansible.rpm already requires these packages.

Reply:
That makes sense. I'll remove it and switch the order of the "oc login" and package installation commands.